test/Monotonicity.lean

```diff
@@ -137,10 +137,10 @@ example {x y z w : ℕ} : true := by
 --     xs ≤ zs := by
 --   revert ys zs
 --   induction' xs with x xs
---   ; intros ys zs h h'
---   ; cases ys with y ys
---   ; cases zs with z zs
---   ; try { cases h ; cases h' ; done },
+--   intros ys zs h h'
+--   cases ys with y ys
+--   cases zs with z zs
+--   try { cases h; cases h'; done },
```

Comment:
I appreciate this is commented out, but removing these semicolons would change meaning, no?

Reply:
This piece of code has not been fully converted to Lean 4 syntax. [The original in mathlib](https://github.com/leanprover-community/mathlib/blob/728ef9dbb281241906f25cbeb30f90d83e0bb451/test/monotonicity.lean#L115C15-L115C15) also uses semicolons, which becomes `<;>` in Lean 4; I've updated the syntax (but not the names).